cmd/machines.go

```diff
@@ -43,11 +44,101 @@ func init() {
 					setUserEnv, checkRequiredFlags, createMachine,
 				),
 			},
+			{
+				Name:  "list",
+				Usage: "List machines for an organization",
+				Flags: []cli.Flag{
+					orgFlag("org to the machine belongs to", true),
+					teamFlag("the team the machine belongs to", false),
+					destroyedFlag(),
```

Comment:
By destroyed what though? The other help text explains that this is specific to destroyed machines.